### ISSUE TITLE
[Triggers] Add a EventListener port-forward command

### ISSUE THREAD

**Issue by dibyom** [tektoncd/cli#1406]
### Feature request

It would be nice to have a `tkn` command to port-forward to an EventListener. 

Currently, users have to use kubectl directly and either have to find the pod that the eventlistener creates or rely on a naming convention that Triggers uses to create EventListener services.

```
kubectl port-forward service/el-my-listener 8080:8080
OR
kubectl port-forward \
 "$(kubectl get pod --selector=eventlistener=github-listener-interceptor -oname)" \
  8000
```

### Use case

* Locally testing/debugging/invoking a Trigger in an EventListener

### UI Example

```
tkn eventlistener port-forward my-listener -n namespace 

tkn eventlistener port-forward my-listener -n namespace -p 9001 #optional port

```

**Comment by chenbh** [comment]
I would be interested to work on this! Do you think it's fair for this to be a very stripped down version of `kubectl` in the sense that only 1 port can be specified and the local port will be the same as the remote port?

**Comment by dibyom** [comment]
Yes, I do not think it is necessary to support all the different options that kubectl supports. However, I think we should support the different local port options to prevent collisions in case the user has something running on that local port already.
